distribution/src/Directory.Build.props

```diff
@@ -6,20 +6,67 @@
   -->
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory).., Directory.Build.props))\Directory.Build.props" />
 
-  <PropertyGroup Label="Nuget package metadata">
-    <OrganizationName><!-- Insert organization name here --></OrganizationName>
-    <RepositoryName><!-- Insert repository name here --></RepositoryName>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+  <!--
+  Place this in your csproj file for each nuget package that needs to be generated.
+
+  <PropertyGroup>
+    <PackageId>my-id</PackageId>
+    <PackageTags>tag1;tag2</PackageTags>
+    <Description>Package description.</Description>
+  </PropertyGroup>
+
+  -->
+
+  <!-- Global nuget package configurations that should not be needed to change -->
+  <PropertyGroup Label="Global Nuget Package metadata">
     <RepositoryUrl>https://github.com/$(OrganizationName)/$(RepositoryName)</RepositoryUrl>
     <RepositoryType>git</RepositoryType>
-    <PackageProjectUrl>https://github.com/$(OrganizationName)/$(RepositoryName)</PackageProjectUrl>
-    <PackageTags></PackageTags>
-    <Product></Product>
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
-    <EmbedUntrackedSources>true</EmbedUntrackedSources>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/$(OrganizationName)/$(RepositoryName)</PackageProjectUrl>
+    <PackageIcon>icon.png</PackageIcon>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <!-- Ensures package icon is valid -->
+    <None Include="..\..\images\icon.png" Pack="true" PackagePath="\" Visible="false" />
+  </ItemGroup>
+
+  <PropertyGroup Label="Build instructions">
+    <OutputType>Library</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+
+    <!-- creates a regular package and a symbols package -->
```

Comment:
creates -> Creates

Reply:
David, would be a great help if you used the "suggestion" features for these kinds of things.